includes/admin/pages.php

```diff
@@ -225,7 +225,9 @@ public function html()
     		if ($tab_id === $current_tab) {
 
     			echo '<div class="simcal-bg-white simcal-p-[2%] simcal-w-[54%] simcal-rounded-[5px] ">';
-
+    			if ($current_tab == 'feeds') {
```

Comment:
Do we need to check if the plugin is installed?

Reply:
This condition is for content display in on setting page tab, So yes it is required in both the case with or without oauth plugin. this was already used in SC I have just use for my condition